net.sourceforge.pmd.eclipse.plugin/src/main/java/net/sourceforge/pmd/eclipse/runtime/properties/impl/ProjectPropertiesImpl.java

```diff
@@ -293,31 +305,41 @@ public void setNeedRebuild(final boolean needRebuild) {
      * @see net.sourceforge.pmd.eclipse.runtime.properties.IProjectProperties#isRuleSetFileExist()
      */
     public final boolean isRuleSetFileExist() {
-        return getResolvedRuleSetFile().exists();
+        boolean ret = true;
+        for (File f : getResolvedRuleSetFile()) {
+            if (!f.exists()) {
+                ret = false;
+            }
+        }
+        return ret;
     }
 
     /**
      * @see net.sourceforge.pmd.eclipse.runtime.properties.IProjectProperties#getResolvedRuleSetFile()
      */
-    public File getResolvedRuleSetFile() {
+    public List<File> getResolvedRuleSetFile() {
         // Check as project-relative path
-        IFile file = project.getFile(getRuleSetFile());
-        File f = getExistingFileOrNull(file);
-        if (f == null) {
-            // Check as workspace-relative path
-            IWorkspaceRoot workspaceRoot = project.getWorkspace().getRoot();
-            try {
-                IFile workspaceFile = workspaceRoot.getFile(new Path(getRuleSetFile()));
-                f = getExistingFileOrNull(workspaceFile);
-            } catch (IllegalArgumentException e) {
-                // Fall back to below
-            }
+        List<File> files = new ArrayList<File>();
+        for (String ruleSetFile : getRuleSetFile().split(",")) {
```

Comment:
This seems to be backwards compatible, which is good and important - so we don't change the format of the project properties file.